README.md

```diff
@@ -0,0 +1,69 @@
+- 📗 Table of Contents
+- 📖 About the Project
+- 🛠 Built With
+- Key Features
+- 💻 Getting Started
+- Setup
+- Prerequisites
+- Install
+- Usage
+- Run tests
+- Deployment
+- 👥 Authors
+- 🔭 Future Features
+- 🤝 Contributing
+- ⭐️ Show your support
+- 🙏 Acknowledgements
+- ❓ FAQ
+- 📝 License
```

Comment:
- [ ] Please your table of content should link to your section and you can achieve that by following the [new Microverse readme template](https://github.com/microverseinc/readme-template) :blush: